t/range.t

```diff
@@ -0,0 +1,9 @@
+use File::Ignore;
+use Test;
+
+plan 1;
+
+ok File::Ignore.parse(q:to/LIST/);
+    [._]*.s[a-w][a-z]
+    [._]s[a-w][a-z]
+    LIST
```

Comment:
Would be nice to have at least one positive/negative test that this ignores the right things.